osu.Game/Screens/Play/Player.cs

```diff
@@ -584,7 +584,10 @@ private void performUserRequestedSkip()
         /// Seek to a specific time in gameplay.
         /// </summary>
         /// <param name="time">The destination time to seek to.</param>
-        public void Seek(double time) => GameplayClockContainer.Seek(time);
+        public void Seek(double time)
+        {
+            GameplayClockContainer.Seek(time);
+        }
```

Comment:
```suggestion
        public void Seek(double time) => GameplayClockContainer.Seek(time);
```

Minor but can be reverted.

Reply:
Done!